tools/linter_lib/custom_check.py

```diff
@@ -216,6 +216,10 @@
             "good_lines": ["assert.ok(...)"],
             "bad_lines": ["assert(...)"],
         },
+        {
+            "pattern": r"allowHTML",
```

Comment:
Should we make this case-insensitive?

Reply:
Changed to `allowHTML|(?i:data-tippy-allowHTML)`.